src/AirtableClient.php

```diff
@@ -111,17 +111,47 @@ public function findTheLatest(string $table, $field, ?string $dataClass = null):
         return AirtableRecord::createFromRecord($recordData);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function addOneRecord(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
+    {
+        $url = sprintf(
+            '%s/%s',
+            $this->id,
+            $table
+        );
+        $response = $this->request($url, 'POST', $fields);
+
+        $recordData = $response->toArray() ?? null;
```

Comment:
I don't get this condition, I think you can remove it `$recordData = $response->toArray();` is enough